### PR TITLE
Reduce corner radius on location bar icon

### PR DESCRIPTION
**Pull request by nullhook**
Resolves https://github.com/brave/brave-browser/issues/16290 - This fixes corner radius on the location bar to match with with the standard corner radius (4px). 

An advanced patch could not be done because the base class definition in chromium code performs a `static_cast` here: https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc;l=131-133 - Overriding the method would still call the base class `GetHighlightPath`. [Here](https://replit.com/@petemill/CommonWastefulMigration#main.cpp) is a sample test.

image

Opted in for a basic patch instead.

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: patches/chrome-browser-ui-views-location_bar-icon_label_bubble_view.cc.patch
Comment (petemill) [inline patches/chrome-browser-ui-views-location_bar-icon_label_bubble_view.cc.patch:9]: This should only contain an additional line, not a removal of the blank line also which it looks like it has.

File: chromium_src/chrome/browser/ui/views/location_bar/icon_label_bubble_view.h
Comment (petemill) [inline chromium_src/chrome/browser/ui/views/location_bar/icon_label_bubble_view.h:9]: hmm, we're patching the .cc file but we're overriding the .h file. So I'm not sure how this works? We need to be providing the macro definition in the .cc file.

ah, it works because we're not `#undef` the macro, so it's staying around forever 😄 

So I think the code is good but we'll need to move it to `icon_label_bubble_view.cc` and call the macro `BRAVE_GET_HIGHLIGHT_PATH` (and make sure it gets undefined too).

**Review by petemill — CHANGES_REQUESTED** [review]

File: chromium_src/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
Comment (mkarolin) [inline chromium_src/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc:7]: Include guards aren't needed in .cc files

File: chromium_src/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
Comment (mkarolin) [inline chromium_src/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc:9]: BRAVE_ICON_LABEL_BUBBLE_VIEW_GET_HIGHLIGHT_PATH would be preferable

File: chromium_src/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
Comment (mkarolin) [inline chromium_src/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc:21]: Same as above: not needed.

**Review by bsclifton — APPROVED** [review]
Changes LGTM, pending feedback flagged by @mkarolin; here's how it looks like on Windows:
image

Nice work 😄

**Review by mkarolin — APPROVED** [review]
LGTM

**Comment by simonhong** [comment]
@nullhook FYI, we set milestone tag after merged.

File: chromium_src/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
Comment (petemill) [inline chromium_src/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc:8]: I'm wondering if this non-centering will be noticeable. We can always remove if it is in the real world in Nightly.

**Review by petemill — APPROVED** [review]
nice and clean!

**Pull request merged**
